tests/test_controller.py

```diff
@@ -1683,44 +1690,57 @@ def collection_link(entry):
             [link] = [l for l in entry['links'] if l['rel']=='collection']
             return link['title'], link['href']
 
-        # One book is in the recommendations feed.
-        [e1] = [e for e in feed['entries'] if e['title'] == self.english_2.title]
-        title, href = collection_link(e1)
-
-        eq_("Recommended Books", title)
+        # This feed contains five books: one recommended,
+        # one in the same series, and two by the same author.
```

Comment:
While fixing this test (which used to use a Work that's no longer created) I changed the way this test works in a way that hopefully makes it clearer where we are getting information about which books are in which recommendation feeds.